website/content/docs/concepts/security/permissions/index.mdx

```diff
@@ -10,69 +10,43 @@ description: |-
 ## Overview
 
 Boundary's permissions model is a composable, RBAC, allow-only model that
```

Comment:
Received feedback that this statement may be confusing, especially for an opening sentence in an overview, will edit